Interfaces/API/Dirac.py

```diff
@@ -75,6 +81,39 @@ def __init__( self, withRepo = False, repoLocation = '', useCertificates = False
 
     self.__clients = None
 
+  def __checkFileArgument( self, fnList, prefix = None, single = False ):
```

Comment:
I didn't know about that practice using isinstance(). What do we do with integers? should we check for int and long? or is it only for some types?